other/add-default-resources/kyverno-test.yaml

```diff
@@ -10,13 +10,12 @@ results:
     patchedResource: patchedResource1.yaml
     kind: Pod
     result: pass
-# nginx-demo2 will get skip as resource already has memory and cpu requests
   - policy: add-default-resources
     rule: add-default-requests
     resource: nginx-demo2
     patchedResource: patchedResource2.yaml
```

Comment:
You may need to remove this.

Reply:
done.

Reply:
Still not succeeding. When you test locally, what is the result? It has to be either `skip` or `pass`.

Reply:
Here's the result I got when it's `skip`:
```
Executing add-default-resources...
applying 1 policy to 3 resources... 

skipped mutate policy add-default-resources -> resource default/Pod/nginx-demo2
│────│───────────────────────│──────────────────────│──────────────────│────────│
│ ID │ POLICY                │ RULE                 │ RESOURCE         │ RESULT │
│────│───────────────────────│──────────────────────│──────────────────│────────│
│  1 │ add-default-resources │ add-default-requests │ /Pod/nginx-demo1 │ Pass   │
│  2 │ add-default-resources │ add-default-requests │ /Pod/nginx-demo2 │ Pass   │
│  3 │ add-default-resources │ add-default-requests │ /Pod/nginx-demo3 │ Pass   │
│────│───────────────────────│──────────────────────│──────────────────│────────│

Test Summary: 3 tests passed and 0 tests failed
```